RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py

```diff
@@ -74,3 +74,10 @@
 phase2_common.toModify(reducedEgamma, 
         preshowerEcalHits = cms.InputTag(""),
 )
+
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+run2_miniAOD_80XLegacy.toModify(
+    reducedEgamma, 
+    ootPhotonPFClusterIsoSources = cms.VInputTag(cms.InputTag("ootPhotonEcalPFClusterIsolationProducer")),
+    ootPhotonPFClusterIsoOutput = cms.vstring("ootPhoEcalPFClusIso")
```

Comment:
I wasn't sure how to do this, so I made with the type specification -- thanks for the example!  done.